dbt_project.yml

```diff
@@ -14,7 +14,7 @@ profile: "aspects"
 model-paths: ["models"]
 analysis-paths: ["analyses"]
 test-paths: ["tests"]
-seed-paths: ["seeds"]
+seed-paths: ["seeds","seeds-test"]
```

Comment:
This one I'm not sure how we would shoehorn into the Aspects context, we obviously don't want these seeds loaded outside of CI. We may need to find / add a way to override the dbt_project.yml in Aspects. 🤔